reconcile/dashdotdb_slo.py

```diff
@@ -84,6 +87,7 @@ def _get_service_slo(self, slo_document):
                     "service": ns['app'],
                     "value": slo_value,
                     "target": slo_target,
+                    "SLODoc": {"name": slo_document['name']},
```

Comment:
The key name is quite inconsistent with the other ones in this dictionary, how about `slo_document`?

Reply:
https://github.com/app-sre/qontract-reconcile/blob/307e426e50595300b0b5b359d761e3d7d078a432/reconcile/dashdotdb_slo.py#L82-L91

It seems pretty consistant with `SLIType` no?